src/main/java/seedu/address/logic/commands/SearchCommand.java

```diff
@@ -2,30 +2,40 @@
 
 import static java.util.Objects.requireNonNull;
 
+import java.util.function.Predicate;
+
 import seedu.address.commons.util.ToStringBuilder;
 import seedu.address.logic.Messages;
 import seedu.address.model.Model;
-import seedu.address.model.person.NameContainsKeywordsPredicate;
+import seedu.address.model.person.Person;
 
 /**
  * Finds and lists all persons in address book whose name contains any of the argument keywords.
  * Keyword matching is case insensitive.
  */
-public class FindCommand extends Command {
+public class SearchCommand extends Command {
 
-    public static final String COMMAND_WORD = "find";
+    public static final String COMMAND_WORD = "search";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all persons whose names contain any of "
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Searches all persons whose names or tags contain any of "
             + "the specified keywords (case-insensitive) and displays them as a list with index numbers.\n"
-            + "Parameters: KEYWORD [MORE_KEYWORDS]...\n"
-            + "Example: " + COMMAND_WORD + " alice bob charlie";
+            + "At least one of the following parameters must be provided.\n"
+            + "Parameters: [n/NAME_KEYWORD] [t/TAG_KEYWORD]\n"
```

Comment:
As per the DG and the other classes, items in square brackets are optional. However, in this case, the keywords are not both optional. As such, using the square brackets in this manner may not be appropriate.

Let me know what you think 👍

Reply:
Thank you Minlong for pointing that out! Which do you think is better, the first or second option? 

Option 1:
```
            + "Parameters: n/NAME_KEYWORD t/TAG_KEYWORD\n"
            + "You can search by name, tag, or both."
            + " If both are provided, only persons matching both criteria will be shown.\n"
            + "Example 1: " + COMMAND_WORD + " n/alice\n"
            + "Example 2: " + COMMAND_WORD + " t/friend\n"
            + "Example 3: " + COMMAND_WORD + " n/alice t/friend\n"
            + "Example 4: " + COMMAND_WORD + " t/friend n/alice";
```

Option 2: 
```

            + "Parameters: /PREFIX KEYWORD [/ANOTHER_PREFIX KEYWORD] \n"
            + "You can search by name, tag, or both."
            + " If both are provided, only persons matching both criteria will be shown.\n"
            + "Example 1: " + COMMAND_WORD + " n/alice\n"
            + "Example 2: " + COMMAND_WORD + " t/friend\n"
            + "Example 3: " + COMMAND_WORD + " n/alice t/friend\n"
            + "Example 4: " + COMMAND_WORD + " t/friend n/alice";
```

Reply:
The second one looks better to me 💯 

If possible, I think the entire message should follow the format:
```
COMMAND_WORD  + "<description>.\n" + 
"Parameters: " + "<parameters> \n" +
"Example: " + "<example>"
```

Reply:
Great! I will go ahead with Option 2 👯‍♂️